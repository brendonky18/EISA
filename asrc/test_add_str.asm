ADD R1, R1, 0x14
ADD R2, R2, 0x1e
ADD R3, R1, R2
STR R3, #0x2d
END

