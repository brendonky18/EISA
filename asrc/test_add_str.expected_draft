000001 00001 00001 1 000000000010100
000001 00010 00010 1 000000000011110
000001 00011 00001 0 00010 0000000000
001110 00011 00000 1 000000000101101
10000000000000000000000000000000
