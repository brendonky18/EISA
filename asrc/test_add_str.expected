00000100001000011000000000010100
00000100010000101000000000011110
00000100011000010000100000000000
00111000011000001000000000101101
10000000000000000000000000000000
